src/main/java/uk/gov/hmcts/reform/sscs/ccd/service/ReadCcdCaseService.java

```diff
@@ -21,26 +19,22 @@
 
     @Autowired
     public ReadCcdCaseService(IdamService idamService,
-                                CcdClient ccdClient,
+                              CcdClient ccdClient,
                               SscsCcdConvertService sscsCcdConvertService) {
         this.idamService = idamService;
         this.ccdClient = ccdClient;
         this.sscsCcdConvertService = sscsCcdConvertService;
     }
 
-    @Retryable
     protected SscsCaseDetails getByCaseId(Long caseId, IdamTokens idamTokens) {
-        log.info("Get getByCaseId " + caseId);
-
+        log.info("*** case-loader *** searching cases by ccdID {}", caseId);
```

Comment:
Can we remove the word case-loader please? Any app can use this method. Maybe pass it in as a parameter if you want to know which app is calling it?

Reply:
sure